src/test/java/racingcar/model/RacingTest.java

```diff
@@ -0,0 +1,77 @@
+package racingcar.model;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.RandomNumberGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RacingTest {
+    //private final List<String> carNames = List.of("박스터", "포르쉐", "소나타")
+    private final String carNames = "박스터, 포르쉐, 소나타";
+
+    @Test
+    void 플레이를_호출하면_모든_자동차_위치_1_증가() {
+        int tryCount = 1;
+        CarFactory carFactory = new CarFactory();
+        List<Car> carList = carFactory.makeCar(carNames);
+
+        Racing racing = new Racing(carList, new AlwaysMoveGenerator());
+        racing.play();
+        List<Integer> positions = new ArrayList<>();
+
+        for(int index = 0; index < carList.size(); index++){
+            positions.add(carList.get(index).getPosition());
+        }
```

Comment:
자바의 stream을 이용하면 다음과 같이 표현할 수 있습니다.
```suggestion
        List<Integer> positions = carList.stream()
                .map(Car::getPosition)
                .collect(Collectors.toList());
```